applications/git.md

```diff
@@ -9,6 +9,64 @@
 ## Getting started
 - [https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
 
+### SSH
+In order to be able to communicate with your repositories via SSH you need to first generate a new SSH key (if you haven't already) and add it to your GitHub, bitBucket etc. account.
+
+#### Generate SSH key
```

Comment:
This could potentially be moved/duplicated to shell.md and/or mac_os.md/linux.md

Opinions @maxrawlinger @ndrsllwngr ?

Reply:
We should try to have always one single source of truth. therefore leaving it here or moving it is fine now. but as we are adding new knowledge we may have to refactor.